tests/reader/test_multipagetiff.py

```diff
@@ -0,0 +1,117 @@
+
+from waveorder.io.multipagetiff import MicromanagerOmeTiffReader
+
+
+def test_constructor_mm2gamma(setup_mm2gamma_ome_tiffs):
+    """
+    test that constructor parses metadata properly
+        no data extraction in this test
+    """
+
+    # choose a specific folder
+    _, one_folder, _ = setup_mm2gamma_ome_tiffs
+    mmr = MicromanagerOmeTiffReader(one_folder, extract_data=False)
+
+    assert(mmr.mm_meta is not None)
+    assert(mmr.width is not 0)
+    assert(mmr.height is not 0)
+    assert(mmr.frames is not 0)
+    assert(mmr.slices is not 0)
+    assert(mmr.channels is not 0)
+    assert(mmr.master_ome_tiff is not None)
+
+
+def test_output_dims_mm2gamma(setup_mm2gamma_ome_tiffs):
+    """
+    test that output dimensions are always (t, c, z, y, x)
+    """
+
+    # choose a random folder
+    _, _, rand_folder = setup_mm2gamma_ome_tiffs
+    mmr = MicromanagerOmeTiffReader(rand_folder, extract_data=True)
+
+    assert(mmr.get_zarr(0).shape[0] == mmr.frames)
+    assert(mmr.get_zarr(0).shape[1] == mmr.channels)
+    assert(mmr.get_zarr(0).shape[2] == mmr.slices)
+    assert(mmr.get_zarr(0).shape[3] == mmr.height)
+    assert(mmr.get_zarr(0).shape[4] == mmr.width)
+
+
+def test_get_zarr_mm2gamma(setup_mm2gamma_ome_tiffs):
+    _, _, rand_folder = setup_mm2gamma_ome_tiffs
+    mmr = MicromanagerOmeTiffReader(rand_folder, extract_data=True)
+    for i in range(mmr.get_num_positions()):
+        z = mmr.get_zarr(i)
+        assert(z.shape == mmr.shape)
+
+
+def test_get_array_mm2gamma(setup_mm2gamma_ome_tiffs):
+    _, _, rand_folder = setup_mm2gamma_ome_tiffs
+    mmr = MicromanagerOmeTiffReader(rand_folder, extract_data=True)
+    for i in range(mmr.get_num_positions()):
```

Comment:
For the get_array and get_zarr tests, do we want to assert that one is a zarr object and one is a numpy array?  This test could pass if the get_array returned a zarr array or vice versa

Reply:
good idea, will do

Reply:
done